kairos-cli/bin/commands/transfer.rs

```diff
@@ -0,0 +1,31 @@
+use crate::common::args::{AmountArg, PrivateKeyPathArg};
+use crate::crypto::public_key::CasperPublicKey;
+use crate::crypto::signer::CasperSigner;
+use crate::error::CliError;
+use crate::utils::parse_hex_string;
+
+use clap::Parser;
+
+type StdVec<E> = Vec<E>;
```

Comment:
What's this alias for?

Reply:
This is not needed in my opinion, especially given that it does not instantiate the generic type. It does not make the code more obvious.

Reply:
It could instead be `type Bytes = std::Vec<u8>`

Reply:
But i would actually say remove it because you need to alias &vec<u8> too then I assume